From 8d4867f792255f86f779723e585a025a17109b69 Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Thu, 02 Mar 2023 09:03:41 +0200
Subject: [PATCH] [SW-116644] gaudi3: include new specs_h9 directory in Makefile

In order to reduce the specs header file sizes CAD team
came with new repo specs_h9. This patch include specs_h9
directory which later will be used to get register
addresses which uses block base instead of config bar
base. This aligns firmware implementation with LKD.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: Ic5de2ee2b1c647424cfe5de5c77b7270fa1960d2
---

diff --git a/Makefile b/Makefile
index f5cb9eb..09148a1 100644
--- a/Makefile
+++ b/Makefile
@@ -18,6 +18,7 @@
 CFLAGS += -I$(SPECS_DIR)/$(BASE_DEV_NAME)/asic_reg
 CFLAGS += -I$(SPECS_EMBEDDED_DIR)
 CFLAGS += -I$(SPECS_EMBEDDED_DIR)/$(SPECS_EMB_DEV)
+CFLAGS += -I$(SPECS_H9_DIR)
 CFLAGS += -I ${EMBEDDED_ROOT}/arcboot/arch
 CFLAGS += -include $(SPECS_EMBEDDED_DIR)/$(SPECS_EMB_DEV)/$(SPECS_EMB_DEV)_reg_map.h
 CFLAGS += -include app/preboot/preboot_app_defs.h
diff --git a/app/ext_rom/Makefile b/app/ext_rom/Makefile
index 177c279..40fb34c 100644
--- a/app/ext_rom/Makefile
+++ b/app/ext_rom/Makefile
@@ -10,6 +10,7 @@
 CFLAGS += -I$(SPECS_DIR)
 CFLAGS += -I$(SPECS_EMBEDDED_DIR)
 CFLAGS += -I$(SPECS_DIR)/$(BASE_DEV_NAME)/asic_reg
+CFLAGS += -I$(SPECS_H9_DIR)
 CFLAGS += -Wall -Werror
 CFLAGS += -g -c
 CFLAGS += -I $(PREBOOT_ROOT_DIR)/include/ -I $(PREBOOT_ROOT_DIR)/ -I $(PREBOOT_ROOT_DIR)/lib/common/ -I $(PREBOOT_ROOT_DIR)/lib/
diff --git a/app/pcie_pif/Makefile b/app/pcie_pif/Makefile
index 476feee..c365a99 100644
--- a/app/pcie_pif/Makefile
+++ b/app/pcie_pif/Makefile
@@ -6,6 +6,7 @@
 HL_DEV_ARCH_COMMON := arch/$(BASE_DEV_NAME)/common
 PREBOOT_ROOT_DIR := ../..
 
+CFLAGS += -I$(SPECS_H9_DIR)
 CFLAGS += -I$(SPECS_DIR)
 CFLAGS += -I$(SPECS_EMBEDDED_DIR)
 CFLAGS += -I$(SPECS_EMBEDDED_DIR)/$(BASE_DEV_NAME)
diff --git a/app/pldm_bloader/Makefile b/app/pldm_bloader/Makefile
index 743c70e..39a4dda 100644
--- a/app/pldm_bloader/Makefile
+++ b/app/pldm_bloader/Makefile
@@ -6,6 +6,7 @@
 HL_DEV_ARCH_COMMON := arch/$(BASE_DEV_NAME)/common
 PREBOOT_ROOT_DIR := ../..
 
+CFLAGS += -I$(SPECS_H9_DIR)
 CFLAGS += -I$(SPECS_DIR)
 CFLAGS += -I$(SPECS_EMBEDDED_DIR)
 CFLAGS += -I$(SPECS_EMBEDDED_DIR)/$(BASE_DEV_NAME)
